### PR TITLE
Detached Vertex package 

### PR DESCRIPTION
**Pull request by zieglerv**
Detached vertex package up to date with development.

### CODE REVIEW

File: reconstruction/vtx/src/main/java/org/jlab/rec/vtx/TrackParsHelix.java [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+	}
+	public void set_p(double _p) {
+		this._p = _p;
+	}
+        public void set_pt(double _pt) {
+		this._pt = _pt;
+	}
+
+	final double LIGHTVEL =  0.0000299792458 ; 
+	public void setHelixParams(double x0, double y0, double z0,
```
Comment (raffaelladevita) [inline reconstruction/vtx/src/main/java/org/jlab/rec/vtx/TrackParsHelix.java:122]: Can we use the helix from clas-tracking?

File: reconstruction/vtx/src/main/java/org/jlab/rec/vtx/VertexFinder.java [hunk 2]
```diff
@@ -0,0 +1,485 @@
...
+                
+	    return v;
+	
+	} // end FindVertex()
+	
+	
+	
+	
+	//
+	//  Interpolation method based on bhabha.f routine.
```
Comment (raffaelladevita) [inline reconstruction/vtx/src/main/java/org/jlab/rec/vtx/VertexFinder.java:129]: Should this commented code be preserved?

**Review by raffaelladevita — CHANGES_REQUESTED** [review]

**Comment by baltzell** [comment]
does this address #774 ?

**Comment by zieglerv** [comment]
Yes.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: Nathan Baltzell ***@***.***>
Sent: Wednesday, March 16, 2022 5:17:41 PM
To: JeffersonLab/clas12-offline-software ***@***.***>
Cc: Veronique Ziegler ***@***.***>; Author ***@***.***>
Subject: [EXTERNAL] Re: [JeffersonLab/clas12-offline-software] Detached Vertex package (PR #799)


does this address #774<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_JeffersonLab_clas12-2Doffline-2Dsoftware_issues_774&d=DwMCaQ&c=CJqEzB1piLOyyvZjb8YUQw&r=t_5zl8Hui3aLne-5dkKLKzPqGCJo_8DZynRDAoZWnLo&m=IHBSdgyU7tetnOkJb_YhamF_a-OVHSCih3up3uujBqYci9mS8B-ivz2nnpWxTbHV&s=KKrQxSA8NEptLL37HIXCQfuLzBn-7LIqTau3eUOdkUQ&e=> ?

—
Reply to this email directly, view it on GitHub<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_JeffersonLab_clas12-2Doffline-2Dsoftware_pull_799-23issuecomment-2D1069643882&d=DwMCaQ&c=CJqEzB1piLOyyvZjb8YUQw&r=t_5zl8Hui3aLne-5dkKLKzPqGCJo_8DZynRDAoZWnLo&m=IHBSdgyU7tetnOkJb_YhamF_a-OVHSCih3up3uujBqYci9mS8B-ivz2nnpWxTbHV&s=eU_eRyjWiAXKYBE-iTi5T8WNr9UaOX2ekUp4lKoXpaw&e=>, or unsubscribe<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_ABYQQB7ZU7B2GWSYVQOV52DVAJFXLANCNFSM5PCXFWKA&d=DwMCaQ&c=CJqEzB1piLOyyvZjb8YUQw&r=t_5zl8Hui3aLne-5dkKLKzPqGCJo_8DZynRDAoZWnLo&m=IHBSdgyU7tetnOkJb_YhamF_a-OVHSCih3up3uujBqYci9mS8B-ivz2nnpWxTbHV&s=vNgNJSSHS_ymPrL1RxPjy1O9qWTKki8aMCqXBp2G5mQ&e=>.
Triage notifications on the go with GitHub Mobile for iOS<https://urldefense.proofpoint.com/v2/url?u=https-3A__apps.apple.com_app_apple-2Dstore_id1477376905-3Fct-3Dnotification-2Demail-26mt-3D8-26pt-3D524675&d=DwMCaQ&c=CJqEzB1piLOyyvZjb8YUQw&r=t_5zl8Hui3aLne-5dkKLKzPqGCJo_8DZynRDAoZWnLo&m=IHBSdgyU7tetnOkJb_YhamF_a-OVHSCih3up3uujBqYci9mS8B-ivz2nnpWxTbHV&s=y3No8siKze0qsibPANR33iNdxzQ13BdftYvJtjTyTtw&e=> or Android<https://urldefense.proofpoint.com/v2/url?u=https-3A__play.google.com_store_apps_details-3Fid-3Dcom.github.android-26referrer-3Dutm-5Fcampaign-253Dnotification-2Demail-2526utm-5Fmedium-253Demail-2526utm-5Fsource-253Dgithub&d=DwMCaQ&c=CJqEzB1piLOyyvZjb8YUQw&r=t_5zl8Hui3aLne-5dkKLKzPqGCJo_8DZynRDAoZWnLo&m=IHBSdgyU7tetnOkJb_YhamF_a-OVHSCih3up3uujBqYci9mS8B-ivz2nnpWxTbHV&s=qzIXPTUqYYy_B4suQJ2TOO0ui7HHdl7ABArwSlvSnEQ&e=>.

**Review by baltzell — DISMISSED** [review]
That new `reconstruction/vtx/pom.xml` appears to inherit some unnecessary, copy-pasted stuff that I think was cleaned up a while back in the development branch for all projects.  Will be good to compare to the others and follow suit, but that can be done later.

Any good reason to hardwire to the "default" CCDB variation?  It's the default, after all.

Also, should use the logger instead of System.out.

**Review by raffaelladevita — APPROVED** [review]

**Pull request merged**